src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -50,22 +54,20 @@ public void armSpeed(double speed) {
     public boolean isInBound(double speed){
```

Comment:
The below bounds angles should go into the constants file as well

Reply:
Done ✅